app/frontend/src/features/friends/FriendRequestsSent.tsx

```diff
@@ -0,0 +1,57 @@
+import {
+  Box,
+  Card,
+  CircularProgress,
+  makeStyles,
+  Typography,
+} from "@material-ui/core";
+import React from "react";
+import Alert from "../../components/Alert";
+import { useIsMounted, useSafeState } from "../../utils/hooks";
+import useFriendsBaseStyles from "./useFriendsBaseStyles";
+
+const useStyles = makeStyles((theme) => ({
+  circularProgress: {
+    display: "block",
+    margin: `0 auto ${theme.spacing(1)}`,
+  },
+  container: {
+    "& > :last-child > $friendItemContent": {
+      marginBottom: theme.spacing(1),
+    },
+  },
+  errorAlert: {
+    borderRadius: 0,
+  },
+  header: {
+    marginLeft: theme.spacing(1),
+    marginBottom: theme.spacing(2),
+    fontWeight: theme.typography.fontWeightBold,
+  },
+}));
```

Comment:
unused?

Reply:
Ahh yeah forgot to delete this when I made this shared styles, good spot :+1: